Neos.Neos/Classes/View/FusionExceptionView.php

```diff
@@ -105,12 +114,16 @@ public function render()
 
         $currentSiteNode = null;
         if ($contentStreamIdentifier instanceof ContentStreamId) {
-            $currentSiteNode = $this->siteNodeUtility->findCurrentSiteNode(
-                $siteDetectionResult->contentRepositoryId,
-                $contentStreamIdentifier,
-                $dimensionSpacePoint,
-                VisibilityConstraints::frontend()
-            );
+            try {
+                $currentSiteNode = $this->siteNodeUtility->findCurrentSiteNode(
+                    $siteDetectionResult->contentRepositoryId,
+                    $contentStreamIdentifier,
+                    $dimensionSpacePoint,
+                    VisibilityConstraints::frontend()
+                );
+            } catch (\RuntimeException $e) {
```

Comment:
I have seen these empty try/catch blocks make debugging real hard because they might hide an underlying exception (e.g. with persistence) – especially when used with low level SPL exceptions.
Shouldn't `find*()` return a nullable node?